src/config.ts

```diff
@@ -448,6 +461,10 @@ function normalizeConfig(config: SnowpackConfig): SnowpackConfig {
     config.proxy = {} as any;
   }
   const allPlugins = {};
+  // remove leading/trailing slashes
+  config.buildOptions.metadataDir = config.buildOptions.metadataDir
+    .replace(/^(\/|\\)/g, '') // replace leading slash
+    .replace(/(\/|\\)$/g, ''); // replace trailing slash
```

Comment:
Also see here: I figured a lot of people would try `/static/snowpack/` as the option, and wanted to handle that here.